src/common/dataModels/jobs.ts

```diff
@@ -8,6 +8,7 @@ export interface IJobsParams {
 
 export interface IJobsQuery {
   shouldReturnTasks: boolean;
+  shouldReturnAvailableActions?: boolean;
```

Comment:
why this is nullable and "shouldReturnTasks" isn't?
they are supposed to be treated the same
( shouldReturnAvailableActions default as "false"
 shouldReturnTasksdefault as "true"
)

Reply:
so shouldReturnTasks should be nullable as well, thats why they have **default** values